server/lib/topStories.js

```diff
@@ -4,11 +4,13 @@ const R = require('ramda');
 const briefArticleMapper = R.map(briefArticle);
 
 const siteApiEndpoint = process.env.siteApiEndpoint;
-const frontPageId = process.env.frontPageId;
-const mainContentKey = process.env.mainContentKey;
-const apiKey = process.env.apiKey;
+const mainContentKey  = process.env.mainContentKey;
+const apiKey          = process.env.apiKey;
+const frontPageIdUK   = process.env.frontPageId;
```

Comment:
@railsagainstignorance In a prior PR you changed these strings from being env vars to being within the codebase itself as it was shown that they are not env specific. Could we apply that work into the PR? Here is the reference PR: https://github.com/ftlabs/big-ft/pull/11/files#diff-c9b6d92f13b7e31c43665de5383a27f0R20